{{cookiecutter.project_slug}}/pyproject.toml

```
@@ -29,6 +29,9 @@ dynamic = [
 ]
 keywords = [
 ]
+license-files = [
+    "LICENSE.md",
+]
```

```suggestion
license = "{{cookiecutter.license}}"
license-files = [
    "LICENSE.md",
]
```

Assuming `cookiecutter.license` is always a valid [SPDX identifier](https://spdx.org/licenses/). Of the three current options `MIT` and `GPL-3.0` appear to be valid, but it might be that `BSD-3` should instead be `BSD-3-Clause` unless shortening is allowed?

---

So from the [official list](https://spdx.org/licenses) it would suggest `BSD-3` is not valid.

From [the PEP](https://peps.python.org/pep-0639) I see no mention of shortening. So I have renamed to `BSD-3-Clause`.